CRM/Mailing/Selector/Browse.php

```diff
@@ -122,41 +122,60 @@ public function &getColumnHeaders($action = NULL, $output = NULL) {
           'sort' => 'name',
           'direction' => CRM_Utils_Sort::DONTCARE,
         ),
+      );
+
+      if (CRM_Core_I18n::isMultilingual()) {
+        self::$_columnHeaders = array_merge(
+          self::$_columnHeaders,
+          array(
+            array(
+              'name' => ts('Language'),
+              'sort' => 'language',
+              'direction' => CRM_Utils_Sort::DONTCARE,
+            ),
+          )
+        );
+      }
+
+      self::$_columnHeaders = array_merge(
```

Comment:
We could, but columns must be in the proper order because the template do a loop on them. I saw this array_merge stuff in CRM/Contribute/Selector/Search.php... 
I guess the alternative is to do array_slice to insert at the proper position but i'm not sure it will be clearer.